mk/travis.sh

```diff
@@ -21,30 +21,22 @@ printenv
 
 case $TARGET_X in
 aarch64-unknown-linux-gnu)
-  DL_TARGET=aarch64-linux-gnu
-  DL_DIGEST=b9137008744d9009877f662dbac7481d673cdcb1798e727e325a37c98a0f63da
+  sudo apt-get install --no-install-recommends g++-aarch64-linux-gnu gcc-aarch64-linux-gnu libc6-dev-arm64-cross -y
+  export QEMU_LD_PREFIX=/usr/aarch64-linux-gnu
   ;;
-arm-unknown-linux-gnueabi)
-  DL_TARGET=arm-linux-gnueabi
-  DL_DIGEST=1c11a944d3e515405e01effc129f3bbf24effb300effa10bf486c9119378ccd7
+arm-unknown-linux-gnueabihf)
+  sudo apt-get install --no-install-recommends g++-arm-linux-gnueabihf gcc-arm-linux-gnueabihf libc6-dev-armhf-cross -y
+  export QEMU_LD_PREFIX=/usr/arm-linux-gnueabihf
   ;;
 *)
   ;;
 esac
 
-if [[ -n ${DL_TARGET-} ]]; then
+if [[  "$TARGET_X" =~ ^(arm|aarch64) ]]; then
   # We need a newer QEMU than Travis has.
   sudo add-apt-repository ppa:pietro-monteiro/qemu-backport -y
   sudo apt-get update -qq
-  sudo apt-get install binfmt-support qemu-user-binfmt -y
-
-  DL_ROOT=https://releases.linaro.org/components/toolchain/binaries/
-  DL_RELEASE=5.1-2015.08
-  DL_BASENAME=gcc-linaro-$DL_RELEASE-x86_64_$DL_TARGET
-  wget $DL_ROOT/$DL_RELEASE/$DL_TARGET/$DL_BASENAME.tar.xz
-  echo "$DL_DIGEST  $DL_BASENAME.tar.xz" | sha256sum -c
-  tar xf $DL_BASENAME.tar.xz
-  export PATH=$PWD/$DL_BASENAME/bin:$PATH
+  sudo apt-get install --no-install-recommends binfmt-support qemu-user-binfmt -y
```

Comment:
These packages need sudo because we're using my backport that is not whitelisted on travis. I requested the whitelist on travis-ci/apt-source-whitelist#271. I'll add the comment for now and update it to use .travis.yml if/when the whitelist gets approved.